.github/workflows/release.yml

```diff
@@ -40,9 +40,10 @@ jobs:
         with:
           path: /root/.cache/
           key: ${{ runner.os }}-go-build
+      # TODO: use goreleaser github action: https://github.com/goreleaser/goreleaser-action
       - run: |
           go install github.com/goreleaser/goreleaser@v1.7.0
-          make release tag=${{ needs.release-please.outputs.release_name }}
```

Comment:
Removing this since goreleaser doesn't use the version passed in through Makefile